docs/sites.yml

```diff
@@ -4954,17 +4954,6 @@
   built_by: d:code:it
   built_by_url: https://dcodeit.com
   featured: false
-- title: SupremeKun - Street Fashion Marketplace and News
```

Comment:
Why is this removed?

Reply:
It looks like this is not a Gatsby Site anymore. I'd say the removal would better to be in a separate PR rather than pushed into this PR

Reply:
I should've validated 🤦‍♂️

I'll send a revert

Reply:
Done in #12597